### PR TITLE
fix(autoplay): Start playback correctly when an ad ends

### PR DESCRIPTION
**Pull request by Thomasvdam**

### CODE REVIEW

File: src/js/utils/isAdItem.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ * @returns {boolean}
+ */
+export default function isAdItem(item) {
+    // Non media items are certenly not ad items.
+    if (item.type !== 'media' || !item.parallel) {
+        return false;
+    }
+
+    // Search for a vast item.
+    return !!item.parallel.find(parallelItem => parallelItem.type === 'vmap');
```
Comment (ivo-toby) [inline src/js/utils/isAdItem.js:15]: Shouldn't VAST not be added as type as well?

Comment (Thomasvdam) [reply src/js/utils/isAdItem.js:15]: Hmm, I think you're probably right. I reused the solution from Hls and Dash, so I guess we should update those too.

Comment (ivo-toby) [reply src/js/utils/isAdItem.js:15]: @FranklinWaller any thoughts about this?

**Comment by ivo-toby** [comment]
I have updated this branch with another fix, should be tested with this branch in media-googleima : https://github.com/meisterplayer/media-nativehls/tree/fix/play_after_ad

@Thomasvdam please review

**Comment by Thomasvdam** [comment]
Looks good. I can't add a review it as it's my own request.

Should we make a separate issue for the isAditem utility?

**Pull request merged**